gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_2_1/schema.yml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.2.1
+      author: GraviteeSource Team
+      changes:
+        # ################
+        # OrganizationLicense changes
+        # ################
+        - createTable:
+            tableName: ${gravitee_prefix}organization_licenses
+            columns:
+              - column: {name: id, type: nvarchar(64), constraints: { nullable: false, primaryKey: true, primaryKeyName: pk_organization_license } }
```

Comment:
I don't think we need an ID for license (and I don't know how we would generate it). We should have only one license per organization so I think the organization_id can be considered the primary key on this table.

Reply:
Ok. If we take that strategy, Mongo will also need to be updated with the `organizationId` as the id.